ibm/service/satellite/resource_ibm_satellite_cluster.go

```diff
@@ -280,6 +280,12 @@ func ResourceIBMSatelliteCluster() *schema.Resource {
 				Sensitive:   true,
 				Description: "The IBM Cloud Identity and Access Management (IAM) service CRN token for the service that creates the cluster.",
 			},
+			"calico_ip_autodetection": {
```

Comment:
is it only for create too? maybe ApplyOnce  or forcenew?

Reply:
yes, it is only for create, so I'll add ApplyOnce